ledger/acctupdates_test.go

```diff
@@ -1224,7 +1222,7 @@ func BenchmarkLargeMerkleTrieRebuild(b *testing.B) {
 	cfg := config.GetDefaultLocal()
 	cfg.Archival = true
 	au, _ := newAcctUpdates(b, ml, cfg)
-	defer au.close()
+	// au and ao are closed via ml.Close() -> ml.trackers.close()
```

Comment:
`ao` is no longer a variable.
```suggestion
	// accountUpdates and onlineAccounts are closed via: ml.Close() -> ml.trackers.close()
```